GIT_AND_GITHUB.md

```diff
@@ -89,6 +77,13 @@ discuss other alternatives. Of course, this means you are putting yourself at ri
 of sharing your knowledge and/or learning some new stuff, so please be careful not
 to end up being even more awesome than you are.
```

Comment:
That could be a great improvement! Can you submit a PR with this suggestions? I haven't changed this part/